From d6bc3c29b060ea2deaae7084c1ef23ef7fe22a4b Mon Sep 17 00:00:00 2001
From: Edward Wu <edwardwu@realtek.com>
Date: Thu, 1 Aug 2024 17:24:03 +0800
Subject: [PATCH] linux-5.15: adjust protected memory region

- release TEE area and AVFW meta pool
- no VE3 FW and bootlogo

Change-Id: Icd20750993685f88ec40bd9db983d04d7661d373
---
 arch/arm64/boot/dts/realtek/rtd16xxb.dtsi | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/arch/arm64/boot/dts/realtek/rtd16xxb.dtsi b/arch/arm64/boot/dts/realtek/rtd16xxb.dtsi
index 17e7cbb85ec0..48b4ff754712 100644
--- a/arch/arm64/boot/dts/realtek/rtd16xxb.dtsi
+++ b/arch/arm64/boot/dts/realtek/rtd16xxb.dtsi
@@ -41,12 +41,6 @@ protected_mem: protected_mem@40000 {
 			no-map;
 		};
 
-		metadata: metadata@c40000 {
-			compatible = "metadata";
-			reg = <0x00c40000 0x003c4000>;
-			no-map;
-		};
-
 		rpc_comm: comm@4080000 {
 			compatible = "comm";
 			reg = <0x04080000 0x1000>;
@@ -57,30 +51,16 @@ rpc_ringbuf: ringbuf@40ff000 {
 			reg = <0x040ff000 0x4000>;
 		};
 
-		video_fw_ve3: video_fw_ve3@4200000 {
-			reg = <0x04200000 0x200000>;
-			no-map;
-		};
-
 		hifi: hifi@f800000 {
 			reg = <0x0f800000 0x800000>;
 			no-map;
 		};
 
-		tee: tee@10000000 {
-			reg = <0x10000000 0x04000000>; /* TEE: 64MB */
-			no-map;
-		};
-
 		vo_reserved: vo_reserved@4800000 {
 			reg = <0x04800000 0x7d000>;
 			no-map;
 		};
 
-		bootlogo_reserved: buffer@2F700000 {
-			reg = <0x2F700000 0x00900000>;
-		};
-
 		cma_reserved_0:linux,cma {
 			compatible = "shared-dma-pool";
 			size = <0x20000000>;			/* 512M */
-- 
2.45.2

